TAO/TAO_IDL/util/utl_global.cpp

```diff
@@ -1930,64 +1930,86 @@ struct yy_buffer_state;
 extern yy_buffer_state *tao_yy_scan_string (const char *);
 extern int tao_yylex_destroy ();
 
+namespace
+{
+  class OldState
+  {
+    public:
+      explicit OldState(bool the_disable_output = false)
+        : old_filename_(idl_global->filename ()),
+          old_lineno_(idl_global->lineno ()),
+          old_idl_src_file_(idl_global->idl_src_file ()),
+          disable_output_(the_disable_output),
+          flags_(ACE_LOG_MSG->flags ())
+      {
+        idl_global->in_eval_ = true;
+
+        idl_global->set_lineno (1);
+        idl_global->set_filename(0);
+
+        // Name this pseudo-file "builtin-N"
+        static char buffer[64];
+        static unsigned n = 1;
+        ACE_OS::snprintf (&buffer[0], sizeof buffer, "builtin-%u", n++);
+        UTL_String utl_string (&buffer[0], true);
+        idl_global->idl_src_file (new UTL_String (&utl_string, true));
+        idl_global->set_filename (new UTL_String (&utl_string, true));
+
+        // Disable Output
+        if (disable_output_)
+        {
+          default_streambuf_ = ACE_DEFAULT_LOG_STREAM->rdbuf ();
+          ACE_DEFAULT_LOG_STREAM->rdbuf (0);
+          ACE_LOG_MSG->clr_flags (ACE_Log_Msg::STDERR);
+          ACE_LOG_MSG->clr_flags (ACE_LOG_MSG->flags ());
+        }
+      }
+
+      ~OldState()
+      {
+        idl_global->set_lineno (old_lineno_);
+
+        // Restore IDL_Global Context
+        idl_global->set_filename (old_filename_);
+        idl_global->idl_src_file()->destroy ();
+        delete idl_global->idl_src_file ();
+        idl_global->idl_src_file (old_idl_src_file_);
+        idl_global->reset_flag_seen ();
+
+        // Renable Output
```

Comment:
```suggestion
```
Either replace "Renable" with "Enable" or "Re-enable", or remove this comment entirely (is it really needed to aid the reader's understanding)?